### ISSUE TITLE
evm t8n is extremely slow

### ISSUE THREAD

**Issue by winsvega** [hyperledger/besu#5229]
I can't batch execute tests on besu's evm cause it takes too long time. been running the whole day not even half the state tests finished. 
is it because besu java is slow on deployment for each individual test? 
in rpc mode was fast.

**Comment by shemnon** [comment]
Java has a huge startup penalty, to JIT all the java classes.  To address this I am working on getting GraalVM builds of the evmtool - #5192

**Comment by winsvega** [comment]
ethereumjs implemented t8n as a service. they start it once and then listener accept http requests and writes response with t8n format to the filesystem

**Comment by shemnon** [comment]
That can be done too.  Can you point me to their interface requirements or what retesteth will need interface wise?

**Comment by shemnon** [comment]
so are there only 2 args or are they quoted or newlined?

**Comment by winsvega** [comment]
it will look like this if to open ${}  bash args
```
curl -X POST -d "--state.fork ArrowGlacier --state.reward 0 --state.chainid 1 --input.alloc /dev/shm/e9bab9d9-8dc8-4812-91bd-4919b0f67552/alloc.json --input.txs /dev/shm/e9bab9d9-8dc8-4812-91bd-4919b0f67552/txs.rlp --input.env /dev/shm/e9bab9d9-8dc8-4812-91bd-4919b0f67552/env.json --output.basedir /dev/shm/e9bab9d9-8dc8-4812-91bd-4919b0f67552 --output.result out.json --output.alloc outAlloc.json" --silent http://localhost:3000/
``

**Comment by shemnon** [comment]
So all inputs and outputs will be via files, and there is no streaming output from the HTTP, it's just CLI args one test per line?

**Comment by non-fungible-nelson** [comment]
Hey @shemnon - any ideas on what to do with this ticket? or tracking progress in a PR elsewhere?

**Comment by winsvega** [comment]
When you have time would be nice to have smth like ethereumjs do. Starting t8n as a daemon accepting requests without class reload. 
Then I will be able to run all the tests we have in reasonable time.

**Comment by winsvega** [comment]
I see besu t8n fails some tests for sure. 
`info: (ValidBlocks/bcExample/optionsTest_London, fork: London, block: 3)` reported different stateRoot

but I can't check it all without the speed/daemon optimization.

**Comment by shemnon** [comment]
Fixed by #5623
